exports/taskfiles/utils/cmake.yaml

```diff
@@ -174,6 +176,8 @@ tasks:
         {{default "" .CMAKE_SETTINGS_DIR}}
       CMAKE_TARGETS:
         ref: "default (list) .CMAKE_TARGETS"
+      CMAKE_SOURCE_DIR: >-
+        {{default "." .CMAKE_SOURCE_DIR}}
```

Comment:
The arguments were alphabetical before the change, so let's keep them alphabetical.